scipy/stats/_continuous_distns.py

```diff
@@ -3542,10 +3542,16 @@ def _pdf(self, r):
         return r * exp(-0.5 * r**2)
 
     def _cdf(self, r):
-        return 1 - exp(-0.5 * r**2)
+        return -special.expm1(-r * r / 2.0)
```

Comment:
@larsmans , Oh I didn't know about the memory implications. Thanks for information. Shall I revert back to the original formula.